pkg/schema/plugins.schema.json

```diff
@@ -11,41 +11,46 @@
       "description": "Version of the plugins schema to which this document adheres",
       "type": "string",
       "default": "1.0.0"
-    }
-  },
-  "required": [
-    "schemaVersion"
-  ],
-  "title": "Plugins json schema",
-  "type": "object",
-  "additionalProperties": {
-    "type": "object",
-    "properties": {
-      "key": {
-        "type": "string"
-      },
-      "value": {
+    },
+    "plugins": {
+      "description": "The definition of plugins to install",
+      "type": "object",
+      "default": "1.0.0",
+      "additionalProperties": {
         "type": "object",
         "properties": {
-          "version": {
-            "description": "The version for the plugins.",
-            "type": "string"
-          },
-          "feedURL": {
-            "description": "The URL of an atom feed where the plugin can be downloaded.",
+          "key": {
```

Comment:
I don't think this is how the properties json schema field works for objects. Have you tested this against the json schema validator (https://www.jsonschemavalidator.net/) with both matching and incorrect schema to make sure it's correct?

Here is how I have defined an object with unknown keys in porter elsewhere (for custom action metadata):

https://github.com/getporter/porter/blob/2aa80a6670bd0aac7284cb8f5ad375b6b44b3365/pkg/schema/manifest.schema.json#L417C6-L421